Dockerfile

```diff
@@ -1,22 +1,22 @@
 FROM jimschubert/8-jdk-alpine-mvn:1.0
 
+COPY docker-entrypoint.sh /usr/local/bin/
```

Comment:
COPY should be after `apk add` otherwise a change to the file would result in not using the cache install of OS packages.